db/migrate/20230404210801_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] As requested by the previous reviewer, The associations between models must be configured correctly according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Since we are using the `author_id` column to reference the modal user, we need to show that the `'author_id'` references the User class. to do this, you need to update the value of the 'foreign_key' attribute to `{ to_table: :users }`. You can write the query like this: 


```ruby
   add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```

Reply:
<img width="1417" alt="Captura de pantalla 2023-04-04 a la(s) 18 29 25" src="https://user-images.githubusercontent.com/99061965/229951472-fad5a99d-3c9d-4cd3-99ca-a7de12c9bbfd.png">